### PR TITLE
MB-10429 Display Pickup address/facility address on Move Task Order tab for NTS

### PR DESCRIPTION
**Pull request by mlm55**
## Summary
Move task order page should display facility address/delivery address instead of “Customer Addresses” for NTS shipments. 

Also edited the dev seed data to make acceptance easier. Removed the `ApprovedDate` from a couple shipments so the UI will use the normal approval endpoint rather than the approve diversion endpoint. Changed `UsesExternalVendor` to false, so the shipments can be approved.

## Setup to Run Your Code

<details>
<summary>💻 You will need to use three separate terminals to test this locally.</summary>

##### Terminal 1

Start the Storybook locally.

```sh
make storybook
```

##### Terminal 2

Start the UI locally.

```sh
make client_run
```

##### Terminal 3

Start the Go server locally.

```sh
make server_run
```

</details>

### Additional steps

1. Run `DEVSEED_SUBSCENARIO=nts_and_ntsr make db_dev_e2e_populate`
2. Login as a a TOO  
3. Look at the `NTSSUB` move and approve both shipments
4. Look at the MTO tab and view that the NTS headers for addresses are Pickup address/Facility address
5. NTSR headers should be Facility address/Delivery address
6.  Look at move `HNTMIS` and approve the HHG shipment
7. Look at the HHG shipment on the MTO tab and verify that the address header still says `Customer's addresses`

## Verification Steps for Author

These are to be checked by the author.

- [ ] Tested in the Experimental environment (for changes to containers, app startup, or connection to data stores)
- [ ] Request review from a member of a different team.
- [ ] Have the Jira acceptance criteria been met for this change?

## Verification Steps for Reviewers

These are to be checked by a reviewer.

### Frontend

- [ ] User facing changes have been reviewed by design.
- [ ] There are no aXe warnings for UI.
- [ ] This works in [Supported Browsers and their phone views](https://github.com/transcom/mymove/tree/master/docs/adr/0016-Browser-Support.md) (Chrome, Firefox, IE, Edge).
- [x] There are no new console errors in the browser devtools
- [x] There are no new console errors in the test output
- [x] If this PR adds a new component to Storybook, it ensures the component is fully responsive, OR if it is intentionally not, a wrapping div using the `officeApp` class or custom `min-width` styling is used to hide any states the would not be visible to the user.

## Screenshots
Screen Shot 2022-01-10 at 1 06 22 PM
Screen Shot 2022-01-10 at 1 06 40 PM
Screen Shot 2022-01-10 at 1 08 04 PM

### CODE REVIEW

File: src/components/Office/ShipmentAddresses/ShipmentAddresses.jsx [hunk 1]
```diff
@@ -22,6 +23,26 @@ const ShipmentAddresses = ({
...
+      destinationHeader = 'Facility address';
+      break;
+    case SHIPMENT_OPTIONS.NTSR:
+      pickupHeader = 'Facility address';
+      destinationHeader = 'Delivery address';
+      break;
+    default:
+      pickupHeader = "Customer's addresses";
+      destinationHeader = '';
+  }
```
Comment (reggieriser) [inline src/components/Office/ShipmentAddresses/ShipmentAddresses.jsx:44]: Nit: Seems like you could initialize the `pickupHeader` and `destinationHeader` to the default case when doing the `let` and get rid of the first and last case.

File: src/components/Office/ShipmentDetails/ShipmentDetailsMain.jsx [hunk 2]
```diff
@@ -76,6 +79,27 @@ const ShipmentDetailsMain = ({
...
+      displayedDeliveryAddress = storageFacility ? storageFacility.address : null;
+      break;
+    case SHIPMENT_OPTIONS.NTSR:
+      displayedPickupAddress = storageFacility ? storageFacility.address : null;
+      displayedDeliveryAddress = destinationAddress;
+      break;
+    default:
+      displayedPickupAddress = pickupAddress;
+      displayedDeliveryAddress = destinationAddress || destinationDutyStationAddress?.postalCode;
+  }
```
Comment (reggieriser) [inline src/components/Office/ShipmentDetails/ShipmentDetailsMain.jsx:101]: Nit: Same thing here with getting rid of two of the cases by initializing the `let` statements.

**Review by reggieriser — APPROVED** [review]
LGTM!  I added a couple of nits (optional to address), but the changes made sense to me and everything worked as I expected when testing it.  You may want to get another approval from another front-end person in case I missed something.

**Review by amanda5iu — APPROVED** [review]
LGTM!

**Pull request merged**